rules-reviewed/eap6/jboss-eap5/jboss-eap5-java.windup.xml

```diff
@@ -44,10 +43,15 @@
                 </javaclass>
             </when>
             <perform>
-                <hint title="javax.jms.QueueConnectionFactory" effort="1" severity="mandatory">
-                    <message>`QueueConnectionFactory` was used to obtain connection to JMS queues.
-                        Replace the lookup string `QueueConnectionFactory` with `ConnectionFactory`.
+                <hint title="javax.jms.QueueConnectionFactory" effort="1" severity="optional">
+                    <message>
+                      <![CDATA[
+                      `QueueConnectionFactory` was used to obtain connection to JMS queues.
+
+                      Replace the lookup string `QueueConnectionFactory` with `ConnectionFactory`.
+                      ]]>
                     </message>
+                    <link title="JBoss EAP 6 - Messaging Configuration" href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html/Administration_and_Configuration_Guide/sect-Configuration1.html#Configure_the_JMS_Server1"/>
```

Comment:
I know it is not everywhere satisfied, but we should keep formatting rules for files like the longest line is 128 characters long -> https://github.com/jboss/ide-config#ide-agnostic-conventions